common/Cargo.toml

```diff
@@ -39,6 +39,5 @@ opentelemetry-semantic-conventions = "0.8.0"
 tracing = "0.1.28"
 
 [dev-dependencies]
-composer = { path = "../composer" }
+composer = { git = "https://github.com/mayadata-io/composer", branch = "develop" }
```

Comment:
Should we use specific composer version, eg a release/0.1 branch or something? Also please make sure building the kubectl static binary image still works. nix-build -A utils.kubectl-binary